Vagrantfile

```diff
@@ -40,26 +44,26 @@ Vagrant.configure("2") do |config|
 
   # Ubuntu 17.04 (Zesty)
   config.vm.define "ubuntu", autostart: false do |ubuntu|
-    config.vm.box = "ubuntu/zesty64"
+    config.vm.box = "generic/ubuntu1704"
     config.vm.provision "shell", inline: "curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -; echo \"deb http://apt.kubernetes.io/ kubernetes-xenial main\" > /etc/apt/sources.list.d/kubernetes.list; apt-get update; DEBIAN_FRONTEND=noninteractive apt-get install -y docker.io golang git qemu-utils selinux-utils systemd-container kubectl tmux"
 
     config.vm.synced_folder ".", "/vagrant", disabled: true
     config.vm.synced_folder ".", "/home/ubuntu/go/src/github.com/kinvolk/kube-spawn",
```

Comment:
I like the change from `ubuntu` from `vagrant`.
But we also need to change `/home/ubuntu` to `/home/vagrant` here in synced_folder. Otherwise rsync could not work.